src/collections/_documentation/platforms/python/bottle.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
I almost overlooked it, you need to add a link to this page in `index.md` (in the same folder) for completeness

Reply:
Added a link. Thanks for noticing.